First_section.html

```diff
@@ -9,53 +9,276 @@
   </head>
 
   <body>
-    <div class="container">
-      <section>
-        <header class="toolbar">
-          <a class="nav" href="#">My Logo</a>
-          <nav><img id="menu" src="union.png" /></nav>
-        </header>
-      </section>
+    <section class="head">
+      <header class="toolbar">
+        <a class="nav" href="#">My Logo</a>
+        <nav><img id="menu" src="union.png" /></nav>
+      </header>
+    </section>
 
-      <main class="content">
+    <main>
+      <section class="intro">
         <h1 id="h1">I am Christian Glad to see you!</h1>
         <p id="paragraph">
           I am software developer! I can help you build a product , feature or
           website Look through some of my work and experience! If you like what
           you see and have a project you need coded, don’t hestiate to contact
           me.
         </p>
-          <h2>LET'S CONNECT</h2>
-       <div class="icons"> 
-        <ul>
-          <li>
-            <a href="#">
-              <img class="social" src="Vector.png" />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social" src="Linkedin icon.png" />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social" src="Vector (1).png" />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social" src="Vector (3).png" />
-            </a>
+        <h2 id="connect">LET'S CONNECT</h2>
+        <div class="icons">
+          <ul class="Image">
+            <li>
+              <a href="#">
+                <img class="social" src="Vector.png" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="social" src="Linkedin icon.png" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="social" src="Vector (1).png" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="social" src="Vector (3).png" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="social" src="Vector (2).png" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="work_container">
+        <div class="work works0">
+          <img src="Snapshoot Portfolio.png" alt="snashoot" />
+          <div class="Project info">
+            <h1 class="project_name">Tonic</h1>
```

Comment:
[Optional]

- [ ] This is important but since the previous reviewer missed this it's marked as optional. Please consider using another `hx` tag for the projects titles other than `h1` there should be only one `h1` tag in a webpage which is the main header of the page 👍

Reply:
Done, Thank you.